.github/workflows/ci-conan-msvc.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Windows MSVC
+
+on: [push, workflow_dispatch]
+
+env:
+  CONAN_UPLOAD: ${{ secrets.CONAN_URL }}
+  CONAN_PASSWORD_SINTEF: ${{ secrets.CONAN_PASSWORD }}
+  CONAN_LOGIN_USERNAME_SINTEF: ${{ secrets.CONAN_USER }}
+  CONFIG_URL: https://github.com/sintef-ocean/conan-configs.git
+
+jobs:
+  conan-with-msvc:
+    name: Conan
+    runs-on: windows-2022
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["1.0"]
```

Comment:
What does this version number refer to?

Reply:
This is the version of cppfmu, which is indicated in version.txt. I know that cppfmu does not have a formal release scheme, but it is useful to have a version in conan. Alternatively, we could adopt YYYY.MM.DD with the latest commit date. Perhaps also add a patch version over the current MAJOR.MINOR.